example/infra/caches.py

```diff
@@ -14,6 +14,10 @@ def __init__(self, ctx: Context) -> None:
         self.cache_root.mkdir(exist_ok=True)
         self.content_store.mkdir(exist_ok=True)
 
+    @property
+    def target_size(self) -> int:
+        return 1024**3
```

Comment:
Is this worth adding as a parameter in the `site.yaml`?

Reply:
Put it here for now as needed to go somewhere... but on the fence about whether it makes sense to go in the site.yaml as the target could be very different on a different machine/setup.

Added to issue #97